### PR TITLE
added first draft SingleValueEncodingContainer doc

### PR DESCRIPTION
**Pull request by turtelneck**
### Description

I created the first draft of a document about the SingleValueEncodingContainer protocol, including a brief description and example. Not sure if the example is a bit overlong, my idea was to create simple, self-contained code.

### Type of Change

N/A

### Checklist

- [ x ] All writings are my own.
- [ x ] My entry follows the Codecademy Docs style guide.
- [ x ] My changes generate no new warnings.
- [ x ] I have performed a self-review of my own writing and code.
- [ x ] I have checked my entry and corrected any misspellings.
- [ x ] I have made corresponding changes to the documentation if needed.
- [ x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swift/concepts/protocols/SingleValueEncodingContainer.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+Title: 'SingleValueEncodingContainer Protocols'
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/SingleValueEncodingContainer.md:2]:
```suggestion
Title: 'SingleValueEncodingContainer'
```

File: content/swift/concepts/protocols/SingleValueEncodingContainer.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Tags:
+  - ['Protocols']()
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A **SingleValueEncodingContainer** protocol is a container that can support the storage and direct encoding of a single non-keyed value. It is part of Swift's `Encoder` framework.
+
+The values you can encode will depend on the chosen encoding format (JSON, plist, etc.).
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/SingleValueEncodingContainer.md:16]:
```suggestion
The values that can be encoded will depend on the chosen encoding format (JSON, plist, etc.).
```

File: content/swift/concepts/protocols/SingleValueEncodingContainer.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## Syntax
+
+```psuedo
+protocol SingleValueEncodingContainer {
+  // Protocol body
+}
+```
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/SingleValueEncodingContainer.md:24]: We typically have a sentence explaining the syntax.

File: content/swift/concepts/protocols/SingleValueEncodingContainer.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## Example
+
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/SingleValueEncodingContainer.md:27]: You need a introductory sentence summarizing the example.

File: content/swift/concepts/protocols/SingleValueEncodingContainer.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+do {
+    let jsonData = try encoder.encode(person)
+    if let jsonString = String(data: jsonData, encoding: .utf8) {
+        print(jsonString)
+    }
+} catch {
+    print(error)
+}
+
+```
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/SingleValueEncodingContainer.md:55]: You need a ` ```shell` block following the example showing the output, like this:

````
This example results in the following output:

```shell
The example's output
```
````

File: content/swift/concepts/protocols/SingleValueEncodingContainer.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+Official Swift Documentation: https://developer.apple.com/documentation/swift/singlevalueencodingcontainer
```
Comment (SSwiniarski) [inline content/swift/concepts/protocols/SingleValueEncodingContainer.md:57]:
```suggestion
```

We don't like to link to off-site documentation in Docs.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@turtelneck I did a first review and left a few comments.

**Comment by caupolicandiaz** [comment]
@turtelneck It has been several weeks and there hasn't been any updates to this entry. Are you still available to finish this?

**Comment by caupolicandiaz** [comment]
@turtelneck @SSwiniarski I'm closing this due to inactivity

**Pull request closed without merging**